GrappleGirlGame/main.lua

```diff
@@ -1,66 +1,100 @@
 require("Character")
 require("love")
-FLOOR_CATAGORY = 1
+FLOOR_CATEGORY = 4
 
 function love.load()
     baseWorld = love.physics.newWorld(0, 1000, false)
-    gGirl = Character:new(nil, baseWorld, {love.graphics.getWidth()/2,100},{400, 400});
-
+    gGirl = Character:new(nil, baseWorld, {love.graphics.getWidth() / 2, 100}, {400, 400})
 
     -- floor --
     floor = {}
     floor.shape = love.physics.newRectangleShape(love.graphics.getWidth(), 100)
-    floor.body = love.physics.newBody( baseWorld, love.graphics.getWidth()/2, love.graphics.getHeight(), "static")
+    floor.body = love.physics.newBody(baseWorld, love.graphics.getWidth() / 2, love.graphics.getHeight(), "static")
     floor.fixture = love.physics.newFixture(floor.body, floor.shape, 1)
-    floor.fixture:setFriction(1);
-    floor.fixture:setCategory(FLOOR_CATAGORY);
+    floor.fixture:setFriction(1)
+    floor.fixture:setCategory(FLOOR_CATEGORY, 10, 9)
```

Comment:
Could you add a few comments to the physics code indicating what some of the calls do? It's hard for me to follow some of it.

Reply:
oops that `10, 9)` was part of my debugging